src/RBControl_battery.cpp

```diff
@@ -119,4 +123,12 @@ uint32_t Battery::rawToMv(uint32_t rawVal) {
     return esp_adc_cal_raw_to_voltage(rawVal, &m_adcChars) * m_coef.load() / BATT_DIVIDER;
 }
 
+void Battery::enableEmergencyShutdown() {
```

Comment:
Could this be an install flag? https://github.com/RoboticsBrno/RB3201-RBControl-library/blob/master/src/RBControl_manager.hpp#L24